crates/router/src/connector/stripe.rs

```diff
@@ -1285,28 +1285,69 @@ impl api::IncomingWebhook for Stripe {
         &self,
         request: &api::IncomingWebhookRequestDetails<'_>,
     ) -> CustomResult<api_models::webhooks::ObjectReferenceId, errors::ConnectorError> {
-        let details: stripe::StripeWebhookObjectId = request
+        let details: stripe::WebhookEvent = request
```

Comment:
Can you please rename `WebhookEvent` as it contains both event_type and event_data?